core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly

```diff
@@ -34,8 +34,9 @@ THE SOFTWARE.
       <l:task href="${rootURL}/" icon="icon-up icon-md" title="${%Back to Dashboard}"/>
       <l:task href="${rootURL}/manage" icon="icon-gear2 icon-md" permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%Manage Jenkins}"/>
       <l:task href="new" icon="icon-new-computer icon-md" permission="${createPermission}" title="${%New Node}"/>
-      <l:task href="${rootURL}/configureClouds" icon="icon-health-40to59 icon-md" permission="${app.ADMINISTER}" title="${%Configure Clouds}"/>
-      <l:task href="configure" icon="icon-gear2 icon-md" permission="${app.MANAGE}" title="${%Node Monitoring}"/>
+      <l:task href="${rootURL}/configureClouds" icon="icon-health-40to59 icon-md" permission="${app.SYSTEM_READ}"
+              title="${app.hasPermission(app.ADMINISTER) ? '%Configure Clouds' : '%View Clouds'}"/>
```

Comment:
Can Overall/Manage users not configure clouds?

Reply:
Not sure... to configure a cloud you need to install a cloud implementation plugin, but for now Overall/Manage is not allowed to install plugins. I'd prefer to delay this decision a bit.

Reply:
No worries, was just confused, that looks like something Manage should be able to and would be very desirable (assuming it's safe).